app/javascript/campaigner_facing/shares_editor.js

```diff
@@ -28,6 +29,29 @@ const SharesEditor = Backbone.View.extend({
     GlobalEvents.bindEvents(this);
   },
 
+  handleError: function(_, response) {
```

Comment:
I'm not a fan of `_` for ignored variables in JavaScript since it's not idiomatic, and it's overriding lodash in this case (which can be confusing).

Reply:
Thanks! Fixed it!